README.md

```diff
@@ -1,10 +1,40 @@
 # Cloud-Optimized Geospatial Formats Guide
 
-See the site [https://developmentseed.org/cloud-optimized-geospatial-formats-guide/](https://developmentseed.org/cloud-optimized-geospatial-formats-guide/)
+---
 
-This site is built using [Quarto](https://quarto.org/docs/get-started/)
-To preview the site locally, install quarto and run:
+**Guide**: <a href="https://tinyurl.com/cogeo-guide" target="_blank">tinyurl.com/cogeo-guide</a>
```

Comment:
Did we want to put this on `ds.io` instead of `tinyurl`?

Also is the reason this is html and not markdown so that you can set `target="_blank"`?

Reply:
What's the full url for ds.io? I think we haven't shortened the url yet.

Reply:
I'm not sure how it works... I think someone has to add the link mapping?

Reply:
I'll try to get to it soon: https://github.com/developmentseed/how/issues/395